src/Pact/Repl.hs

```diff
@@ -19,7 +19,33 @@
 -- functionality.
 --
 
-module Pact.Repl where
+module Pact.Repl
+  ( errToUnit
+  , execScript
+  , execScript'
+  , evalPact
+  , evalRepl
+  , evalRepl'
+  , evalString
+  , getDelta
+  , handleCompile
+  , handleParse
+  , initPureEvalEnv
+  , initReplState
+  , isPactFile
+  , parsedCompileEval
+  , pipeLoop
+  , rSuccess
+  , repl
+  , repl'
+  , runPipedRepl
+  , runPipedRepl'
+  , setReplLib
+  , toUTF8Bytes
+  , trim
+  , unsetReplLib
+  , utf8BytesLength
```

Comment:
@vaibhavsagar did you base this on external usage?

Reply:
Yes, of course. Are there exports you think are unnecessary?

Reply:
No, just shocked how many are used. Thanks!